fahrschein-opentracing/src/main/java/org/zalando/fahrschein/opentracing/InstrumentedPublishingHandler.java

```diff
@@ -0,0 +1,59 @@
+package org.zalando.fahrschein.opentracing;
+
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+import io.opentracing.tag.Tags;
+import org.zalando.fahrschein.http.api.EventPublishingHandler;
+import org.zalando.fahrschein.http.api.Response;
+
+import java.util.List;
+import java.util.Locale;
+
+import static io.opentracing.tag.Tags.ERROR;
+
+public class InstrumentedPublishingHandler implements EventPublishingHandler {
```

Comment:
Yes. The implementation in `...opentelemetry.InstrumentedPublishingHandler` is slightly different because it uses the opentelemetry API instead of the opentracing one.

`sizeBucket` seems to be a common feature of both. <Nit>It could go in the interface as a static or instance method</Nit>.

Reply:
As the `sizeBucket` calculation is specific to the tracing logic I don't see that it should move to the API module. At the same time, I think it's not worth the overhead to introduce a shared module for that.